tests/e2e/e2e_simple_test.go

```diff
@@ -59,7 +59,7 @@ var _ = Describe("ISO build test", func() {
 										Name:    "test",
 										Image:   "debian:latest",
 										Command: []string{"bash"},
-										Args:    []string{"-xec", "[ -f /artifacts/*.iso ]"},
+										Args:    []string{"-xec", "[ -f /artifacts/build/*.iso ]"},
```

Comment:
auroraboot `build-iso` command [creates a `build` subdirectory](https://github.com/kairos-io/AuroraBoot/blob/cc8b857ec7c483c63182659c96c6f7caa0fccd0c/deployer/steps.go#L189). I wonder if we should get rid of that. If the user says `--output mydir` they would expect to see the iso file in `mydir` right? Not in `mydir/build/my.iso`. Depending on what we decide to do, we should also fix [the docs](https://github.com/kairos-io/kairos-docs/blob/98c99f61a03d5c6c0e776c22abdfffb7e6a45cc8/content/en/docs/Advanced/build.md?plain=1#L100).

Reply:
agreed, this also changes behavior from enki

Reply:
Fixed here: https://github.com/kairos-io/AuroraBoot/pull/115